image/templates/helm/shared/templates/_metadata.tpl

```diff
@@ -1,45 +1,73 @@
 {{/*
-  srox.labels $ $objType $objName
+  srox.labels $ $objType $objName [ $labels ]
 
   Format labels for $objType/$objName as YAML.
```

Comment:
Please document the meaning of the new optional arg.

Here and below.

Reply:
done